src/features/payment/PaymentScreen.tsx

```diff
@@ -351,25 +348,17 @@ const PaymentScreen = () => {
         return [false, '']
       }
 
-      if (ticker === 'MOBILE') {
-        // If paying with mobile, they need to have enough mobile to cover the payment
-        // and enough hnt to cover the fee
-        const hasEnoughNetwork =
-          networkBalance.minus(paymentState.networkFee).integerBalance >= 0
-        const hasEnoughMobile =
-          mobileSolBalance - paymentState.totalAmount.floatBalance.valueOf() >=
-          0
-        if (!hasEnoughNetwork) return [true, networkBalance.type.ticker]
-        if (!hasEnoughMobile) return [true, 'MOBILE']
-      }
-
-      if (ticker === 'IOT') {
-        const hasEnoughNetwork =
-          networkBalance.minus(paymentState.networkFee).integerBalance >= 0
-        const hasEnoughIOT =
-          iotSolBalance - paymentState.totalAmount.floatBalance.valueOf() >= 0
-        if (!hasEnoughNetwork) return [true, networkBalance.type.ticker]
-        if (!hasEnoughIOT) return [true, 'IOT']
+      if (l1Network === 'helium' || l1Network === 'solana') {
```

Comment:
When does `l1Network` not equal `helium` or `solana`?

Reply:
Oops should only be checking `helium` here